src/lines/ssrc-group-line.ts

```diff
@@ -0,0 +1,54 @@
+import { NUM, SP } from '../regex-helpers';
+import { Line } from './line';
+
+type SsrcGroupSemantics = 'SIM' | 'FID' | 'FEC';
+
+/**
+ * Model an SSRC group line as define by https://datatracker.ietf.org/doc/html/rfc5576#section-4.2.
+ *
+ * @example
+ * a=ssrc-group: FID 13579 24680
+ */
+export class SsrcGroupLine extends Line {
+  semantics: SsrcGroupSemantics;
+
+  ssrcs: number[];
+
+  private static regex = new RegExp(`^ssrc-group:(SIM|FID|FEC) ((?:${NUM}${SP}*)+)`);
```

Comment:
Technically the spec allows for a SSRC group line with zero SSRC IDs which would fail this regex. Not sure if this is something we should consider.

Reply:
Good catch...no idea what use that is :) I played with this a bit and found it quite annoying to build into the regex.  Considering we support an explicit set of semantics at this point, and none of those support an empty list, I think I'll leave it as-is for now.